packages/frontend/src/pages/Login/index.tsx

```diff
@@ -1,99 +1,49 @@
-import * as URLS from 'config/urls'
-
-export default function Login(): React.ReactElement {
-  const theme = useTheme()
-  const navigate = useNavigate()
-
-  const matchSmallScreens = useMediaQuery(theme.breakpoints.down('md'), {
-    noSsr: true,
-  })
 
+export default function Login(): ReactElement {
   return (
-    <Box
-      sx={{
-        display: 'flex',
-        flex: 1,
-        alignItems: 'stretch',
-        flexDirection: matchSmallScreens ? 'column' : 'row',
-      }}
-    >
-      <Box
+    <Flex flex={1} alignItems="stretch" flexDir={{ base: 'column', md: 'row' }}>
+      {/* Left half of the page: Login graphic */}
+      <Flex
         flex={1}
-        maxWidth="700px"
-        display={matchSmallScreens ? 'none' : 'flex'}
-        color="white"
-        sx={{ backgroundColor: 'primary.light' }}
-        alignItems="center"
-        justifyContent="center"
-        flexDirection="column"
-      >
-        <Typography
-          variant="h2"
-          component="h3"
-          color="primary.main"
-          marginX={10}
-          marginBottom={5}
-        >
-          Build and automate your work<i>flows</i>
-        </Typography>
-        <Box
-          component="img"
-          src={landingImg}
-          width="100%"
-          left="0"
-          paddingRight={10}
-        />
-      </Box>
-      <Box
-        flex={matchSmallScreens ? 2 : 1}
-        width="100%"
-        paddingTop={matchSmallScreens ? '5vh' : 0}
-        sx={{
-          display: 'flex',
-          alignItems: 'center',
-        }}
+        flexDir="column"
+        display={{ base: 'none', md: 'flex' }}
+        bg="primary.50"
+        color="primary.600"
+        pt="3.75rem"
+        gap={{ base: '3rem', lg: '4.5rem' }}
       >
-        <Container maxW="container.sm">
-          <Box
-            display="flex"
-            flexDirection="column"
-            justifyContent="center"
-            alignItems={matchSmallScreens ? 'center' : 'start'}
-          >
-            {matchSmallScreens && (
-              <Box component="img" src={landingImg} width="35vw" mb={3} />
-            )}
-            <Typography
-              variant={'h2'}
-              fontSize={'1.5rem'}
-              noWrap
-              fontWeight="bold"
-              fontFamily={'Space Grotesk'}
-              marginBottom={3}
-              sx={{
-                display: 'flex',
-                alignItems: 'center',
-                cursor: 'pointer',
-                userSelect: 'none',
-              }}
-              onClick={() => navigate(URLS.ROOT)}
-            >
-              <Box component="img" src={mainLogo} height="100%" mr={1} />
-              <FormattedMessage id="brandText" />
-            </Typography>
-          </Box>
-          <LoginForm />
-        </Container>
-      </Box>
-    </Box>
+        <Flex flexDir="column" gap={2} pl={11}>
+          <Text textStyle="heavy" maxW="60%">
+            Automate your manual and repititive tasks
```

Comment:
typo - maybe install a spellchecker in your IDE?
```suggestion
            Automate your manual and repetitive tasks
```

Reply:
ok will do, thanks!